databaseQueries/solutions.js

```diff
@@ -125,7 +125,7 @@ module.exports = class Solutions {
 	 * @returns {Array} list of indexes.
 	 */
 
-	static listIndexesFunc() {
+	static listIndexes() {
```

Comment:
@Prajwal17Tunerlabs check the name is changes every where this function is being used

Reply:
this change will not harm, we can keep the change.